content/docs/overview/about.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "About NuoDB Control Plane"
+description: ""
+summary: ""
+date: 2024-08-14T13:52:09+03:00
+lastmod: 2024-08-14T13:52:09+03:00
+draft: false
+weight: 15
+toc: true
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+NuoDB Control Plane (CP) enables customers to consume NuoDB in a Database as a Service (DBaaS) model.
+It runs inside a Kubernetes cluster and handles provisioning and management of multiple domains, and their databases, in the _same_ cluster.
+NuoDB databases are created on-demand remotely using REST services using various predefined configuration options.
+
+## Run on the Cloud
+
+{{< card-grid >}}
+{{< link-card title="Install NuoDB Control Plane" href="../getting-started/installation.md" icon="outline/stack-push" description="Start managing NuoDB databases deployed in Kubernetes" >}}
+{{< link-card title="Deploy NuoDB databases" href="../getting-started/create-database.md" icon="outline/rocket" description="Create NuoDB databases on-demand easily" >}}
+{{< link-card title="Start Developing" href="../getting-started/connect-database.md" icon="outline/code" description="Connect to your database and start developing with NuoDB" >}}
+{{< /card-grid >}}
+
+## Architecture
+
+The NuoDB Control Plane allows the creation of:
+
+- One or more organizations
+- Each organization can have one or more projects.
+Each project corresponds to a NuoDB
+domain with its own set of NuoDB Admin processes (APs).
+- Within a project you can deploy one or more databases.
```

Comment:
Suggest trying to remove the use of "you" (second person) in the documentation.

e.g., 

"- Within a project, one or more databases can be deployed."